shared/domains/chelonia/chelonia.js

```diff
@@ -39,13 +44,38 @@ export const ACTION_REGEX: RegExp = /^((([\w.]+)\/([^/]+))(?:\/(?:([^/]+)\/)?)?)
 // 4 => 'group'
 // 5 => 'payment'
 
+const signatureFnBuilder = (key) => {
+  return (data) => {
+    return {
+      type: key.type,
+      keyId: sbp('gi.crypto/key/id', key),
+      data: sbp('gi.crypto/sign', key, data)
+    }
+  }
+}
+
 sbp('sbp/selectors/register', {
   // https://www.wordnik.com/words/chelonia
   // https://gitlab.okturtles.org/okturtles/group-income/-/wikis/E2E-Protocol/Framework.md#alt-names
   'chelonia/_init': function () {
     this.config = {
-      decryptFn: JSON.parse, // override!
-      encryptFn: JSON.stringify, // override!
+      decryptFn: (message) => JSON.parse(message), /* (message, state) => {        if (typeof message === 'string') {
+          return JSON.parse(message)
+        }
+        const key = state._volatile?.[message.keyId]
+        if (key) {
+          return JSON.parse(sbp('gi.crypto/decrypt', sbp('gi.crypto/key/deserialize', key), message))
+        }
+      } */
+      encryptFn: (message) => JSON.stringify(message), /* (message, state) => {        const key = state._vm?.authorizedKeys?.find((k) => k.type === 'curve25519xsalsa20poly1305' && Array.isArray(k.perm) && k.perm.includes(GIMessage.OP_ACTION_ENCRYPTED))
+        if (!key) {
+          return JSON.stringify(message)
+        }
+        return {
+          keyId: key.id,
+          content: sbp('gi.crypto/encrypt', sbp('gi.crypto/key/deserialize', key.data), JSON.stringify(message))
+        }
+      } */
```

Comment:
The defaults should be left as they were - simple.

Any overrides should be done outside of this file by calling `/configure`